float64 x
float64 y
float64 theta
float64 v
float64 w
# float64 v_x
# float64 v_y
time timestamp
